src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] infoFromFileName = getInfoFromFileName(fromFileName);
+        String reportSupplyAndBuy = getInfoSupplyAndBuy(infoFromFileName);
+        writeToFile(toFileName, reportSupplyAndBuy);
+    }
+
+    public String[] getInfoFromFileName(String fromFileName) {
```

Comment:
```suggestion
    private String[] readFile(String fileName) {
```
I'd recommend making this declaration more clear. And also pay attention to the access modifiers